src/sider/arbeidsuforhet/RundskrivLenke.tsx

```diff
@@ -0,0 +1,26 @@
+import { Box, Heading } from "@navikt/ds-react";
+import React from "react";
+import { EksternLenke } from "@/components/EksternLenke";
+
+const texts = {
+  heading: "Nyttig lenke",
+  link: "Rundskriv på 8-4 arbeidsuførhet",
```

Comment:
Paragraf-tegn foran 8-4? `Rundskriv på § 8-4 arbeidsuførhet`. Er det rundskriv `for` eller `på` en paragraf?

Reply:
Godt innspill! Jeg legger til paragraftegnet. Usikker på preposisjonen selv, kan høre med Stine! 😊